scripts/jupyter-bundler

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+from notebook.bundlerextensions import main
```

Comment:
It would probably also be good for this entrypoint to be accessible in `notebook.bundler.__init__`, so that one can also launch this with `python -m notebook.bundler`